python/src/fragment.rs

```diff
@@ -433,6 +434,24 @@ impl FragmentMetadata {
         )
     }
 
+    #[getter]
+    fn physical_rows(&self) -> Option<usize> {
```

Comment:
If you document these methods (with rust style docs) it should generate python style docs on the bindings.  Can you do that?  Otherwise a user (even a sophisticated pylance user) will wonder the difference between `physical_rows` and `num_rows`.